### ISSUE TITLE
Filter by ticketType not functional

### ISSUE THREAD

**Issue by RitterKnightCreative** [verbb/Events#105]
### Describe the bug

Trying to do a ticket query by [ticketType](https://verbb.io/craft-plugins/events/docs/getting-elements/purchased-ticket-queries#ticketType) is getting unexpected results.

### Steps to reproduce

For example, doing something like:

```
{% craft.events.purchasedTickets().eventId(event.id).ticketType(['late','standard']).asArray().select('orderId').all %}
```
Seems to only take one of the two types passed in.  Leaving ``ticketType()`` off gets me all tickets as expected.

If I only specify one ``ticketType`` it works as expected.

However, if I specify ``not standard`` or ``['not','standard']`` which I tried initially as well doesn't work.

Unless I'm misunderstanding how this is supposed to work?

### Craft CMS version

3.7.50

### Plugin version

1.4.21

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/events` requirement in `composer.json` to:

```
"require": {
  "verbb/events": "dev-craft-3 as 1.4.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by RitterKnightCreative** [comment]
Fantastic Josh - you guys rock!  Is this slated for a "general" release at some point?

**Comment by engram-design** [comment]
Released in [1.4.22](https://github.com/verbb/events/releases/tag/1.4.22)
